modules/installing-gitops-operator-in-web-console.adoc

```diff
@@ -18,3 +18,7 @@
 Red Hat OpenShift GitOps will be installed in all namespaces of the cluster.
 
 Once the Red Hat OpenShift GitOps Operator is installed, it automatically sets up a ready-to-use Argo CD instance that is available in the `openshift-gitops` namespace, and an Argo CD icon is displayed in the console toolbar.
+
+If you have OpenShift GitOps Operator already installed on your cluster, the existing installation is seamlessly upgraded. 
+
+NOTE: If you're upgrading from v1.0.1, please use the **stable** channel which enables installation of the latest stable and supported release of the Red Hat OpenShift GitOps Operator.
```

Comment:
We use active voice where possible, please change this to: 'If you upgrade'
It also removes the you're contraction which we do not use in the docs. 
Remove the word 'please', we need to be authoritative - as in - you must do x, y, z
Either replace 'which' with 'that' or add a comma after channel. When you use which, it needs to be preceded with a comma.